go/consensus/tendermint/apps/roothash/transactions.go

```diff
@@ -123,7 +154,7 @@ func (app *rootHashApplication) executorCommit(
 	}
 
 	for _, commit := range cc.Commits {
-		if err = rtState.Round.AddExecutorCommitment(ctx, &commit, sv, nl); err != nil { // nolint: gosec
+		if err = rtState.Round.AddExecutorCommitment(ctx, app.state.BlockHeight(), &commit, sv, nl); err != nil { // nolint: gosec
```

Comment:
Use `ctx.BlockHeight()` (since you already pass `ctx` the called method could already access that).